client/cmdlfviking.c

```diff
@@ -17,7 +17,6 @@
 #include "cmddata.h"
 #include "cmdmain.h"
 #include "cmdlf.h"
-#include "cmdlfviking.h"
```

Comment:
I know that you don't like header files,  but removing its own is a bit harsh don't you think?